src/foam/core/Duration.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright 2016 The FOAM Authors. All Rights Reserved.
```

Comment:
@kgrgreer this is the year of the original types.js that Duration was pulled from.